### PR TITLE
Sponsors

### LINKED ISSUE #208: Add sponsor section
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have searched existing issues to ensure the feature has not already been requested

### 🚀 Feature Proposal

We must comply to our sponsorship program:

> Add your logo (max-width: 100px), will be showcased on the official website
> Add your logo (max-width: 160px), will be showcased on the official website

### Motivation

We have our first tier 3 sponsor: 

https://github.com/fastify/fastify/pull/5424

Will work on it this weekend

### Example

_No response_

### PR DESCRIPTION
**Pull request by Eomm**
closes #208 

If you want a different layout, please provide precise instructions or colour.

image

Logo source: https://commons.wikimedia.org/wiki/File:Mercedes-Benz_Star_2022.svg

I should normalize the `Acknowledgments`'s sponsors because they sponsor us with development time.

### CODE REVIEW

**Comment by climba03003** [comment]
The logo doesn't seems correct. It is not centered inside the viewbox and shrink-ed.
image

**Comment by Eomm** [comment]
> The logo doesn't seems correct. It is not centered inside the viewbox and shrink-ed. ![image]

The SVG was a mess - replaced with the png.
I spent too much time looking for an official svg image

image

File: src/pages/index.jsx [hunk 1]
```diff
@@ -40,11 +40,33 @@ export default function Home() {
...
-              to <Link to="/organisations#how-to-be-featured-here">be featured here</Link>?
-            </p>
-            <Organisations maxItems={12} />
+          </div>
+
+          <div className="container">
+            <div className="row">
+              <div className="col col--6">
+                <div className="alternate-sponsors">
+                  <h3>Sponsoring</h3>
```
Comment (mcollina) [inline src/pages/index.jsx]:
```suggestion
                  <h3>Sponsors</h3>
```

**Review by mcollina — CHANGES_REQUESTED** [review]

**Comment by mcollina** [comment]
Please remove the yellow background

**Comment by Eomm** [comment]
Fixed:

- removed yellow bg
- implemented the tier max width for the images

File: src/pages/index.jsx [hunk 2]
```diff
@@ -40,11 +40,33 @@ export default function Home() {
...
+                    <Link to="https://github.com/sponsors/fastify">GitHub</Link> or{' '}
+                    <Link to="https://opencollective.com/fastify">Open Collective</Link>.
+                  </p>
+                  <Organisations onlySponsors={true} />
+                </div>
+              </div>
+
+              <div className="col col--6">
+                <div className="alternate">
+                  <h3>Using</h3>
```
Comment (mcollina) [inline src/pages/index.jsx:61]:
```suggestion
                  <h3>Who is using Fastify?</h3>
```

Comment (Eomm) [reply src/pages/index.jsx:61]: It is already there:

image

Comment (mcollina) [reply src/pages/index.jsx:61]: Ah, then we should move sponsors into its own section.

Comment (Eomm) [reply src/pages/index.jsx:61]: > If you want a different layout, please provide precise instructions or colour.

**Review by fox1t — APPROVED** [review]

File: src/pages/index.jsx (hunk 2 above)
Comment (Eomm) [reply src/pages/index.jsx:61]: can we land this and then improve it?

Comment (mcollina) [reply src/pages/index.jsx:61]: @Fdawgs can you suggest some
alternative copy/structure here?

File: src/pages/index.jsx [hunk 3]
```diff
@@ -40,11 +40,33 @@ export default function Home() {
...
-            <Organisations maxItems={12} />
+          </div>
+
+          <div className="container">
+            <div className="row">
+              <div className="col col--6">
+                <div className="alternate-sponsors">
+                  <h3>Sponsors</h3>
+                  <p>
+                    Would you like to sponsor Fastify financially? Support us on{' '}
```
Comment (Fdawgs) [inline src/pages/index.jsx:51]:
```suggestion
                    Would you like to sponsor Fastify? Support us on{' '}
```
I think `financially` is redundant as we're already asking if the user if they would like to sponsor Fastify, which implies financial aid.

Comment (Eomm) [reply src/pages/index.jsx:51]: The issue I wanted to solve is that we have these sponsors:

https://github.com/fastify/fastify?tab=readme-ov-file#sponsors

but they sponsored fastify with devs' time and not money.

I think we should normalize that section TBH

cc @mcollina

Comment (gurgunday) [reply src/pages/index.jsx:51]: I think both should certainly exist but there should be a clear distinction between pure financial support to Fastify team and all other forms of support such as dev time etc.

**Review by gurgunday — APPROVED** [review]

**Comment by gurgunday** [comment]
I think we can merge and improve in the following PRs

**Review by Fdawgs — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**